queue_job/jobrunner/runner.py

```diff
@@ -398,10 +398,16 @@ def from_environ_or_config(cls):
         return runner
 
     def get_db_names(self):
+        """
+        >>> runner = QueueJobRunner()
+        >>> config["db_name"] = None
+        >>> runner.get_db_names()
+        ['odoo']
+        """
```

Comment:
What about add `list_db = False` to test `odoo.service.db.list_dbs(True)` against `odoo.service.db.exp_list(True)`?
```
>>> config["list_db"] = False
```

Reply:
Yes, that's much better

Reply:
oh, i finally understood. Yes, thats a great idea, and a much better test approach
This only checks list_dbs gives something like untested exp_lists, your test checks if it works without list_dbs

Reply:
:) Please see https://github.com/OCA/queue/pull/591